ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerFile/main.qml

```diff
@@ -0,0 +1,12 @@
+import QtQuick 2.6
+import QtQuick.Controls 2.2
```

Comment:
not needed?

Reply:
Turns out is needed for the ApplicationWindow dependency. I was surprised myself. 
http://doc.qt.io/qt-5/qml-qtquick-controls-applicationwindow.html